gcloud/pubsub/test_client.py

```diff
@@ -44,7 +81,7 @@ def test_list_topics_no_paging(self):
         self.assertEqual(next_page_token, None)
 
         self.assertEqual(len(conn._requested), 0)
```

Comment:
Why would `conn._requested` have no entries? Also, why do we need a mock connection in this test case after the change?